server/routes/filmsRoutes.js

```diff
@@ -0,0 +1,52 @@
+const mongoose = require('mongoose');
+const Film = mongoose.model('films');
+const User = mongoose.model('users');
+
+module.exports = (app) => {
+  app.get(`/api/films/:pageId/`, async (req, res) => {
+    const { pageId } = req.params
+    i = 5
+    y = i * (pageId - 1)
+    const films = await Film.find().skip(y).limit(i);
+
+    return res.status(200).send(films);
+  });
+
+  app.get(`/api/film/title`, async (req, res) => {
+    const films = await Film.find({ "title": { "$regex": req.query.keyword }});
+
+    return res.status(200).send({
+      error: false,
+      films,
+    });
+  });
+
+  app.get(`/api/film/actor`, async (req, res) => {
+    const { keyword } = req.query;
+    const film = await Film.find({ 'actor': { "$regex": keyword }});
+
+    return res.status(202).send({
+      error: false,
+      film,
+    });
+  });
+
+  app.post(`/api/films`, async (req, res) => {
+    const film = await Film.create(req.body);
+
+    return res.status(201).send({
+      error: false,
+      film,
+    });
+  });
+
+  app.delete(`/api/films/:filmId`, async (req, res) => {
+    const { id } = req.params;
```

Comment:
It looks like there could be a typo.
You defined a `filmId` on
[line 43](https://github.com/jbryant92/black-codher-personal-project/pull/1/files#diff-667455c95b33412ef6c29b2f3d6ce1b342cc8af89f918006b885d244bb4249f8R43) 
but you are referencing id in the params lookup which will always result in it being null
```suggestion
    const { filmId } = req.params;
```